### DISCUSSION TITLE
Sending parameters when the component is rendered

### DISCUSSION THREAD

**Discussion by 1YalcinSahin1** [livewire/livewire#4792] [1. Help]
```
class AdminContent extends Component
{
    public array $integrations;

    public function render(): Factory|View|Application
    {
        return view('livewire.admin-content')->with([
            "integrations" => ($this->integrations)
        ]);
    }
}
```

```
  @livewire('admin-content',['integrations' => $integrations])
                    
<livewire:admin-content :integrations="$integrations" />
```
These uses work on windows. But second usage doesn't work on Ubuntu(18.04).

if the parameter does not start with the letter 'i' it works. Example:

`<livewire:admin-content :ntegrations="$integrations" />`

It is working.

**Comment by joshhanley** [comment]
@1YalcinSahin1 hmm that's strange. Sounds like a configuration issue on Ubuntu, but I wouldn't know where to start to troubleshoot that. I use Ubuntu for my servers and don't have any issues with it.

**Reply by 1YalcinSahin1** [reply]
I encountered this error after upgrading Laravel 9 and php 8.1.I don't know much about Ubuntu.So I had to make the first use.
